.github/workflows/backend-pplnn.yml

```diff
@@ -17,23 +17,33 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  script_install:
-    runs-on: ubuntu-18.04
-    strategy:
-      matrix:
-        python-version: [3.7]
+  build_pplnn:
+    runs-on: [self-hosted, linux-3090]
```

Comment:
why on `self-hosted`?

Reply:
The old workflow takes around 30min and is too slow. We have pplnn built on the custom image and this could speed up the process.